project_files/xcode/hector.xcodeproj/project.pbxproj

```diff
@@ -7,9 +7,6 @@
 	objects = {
```

Comment:
@bpbond and @rplzzz do we want to commit the xcode project file?

Reply:
You probably should.  Windows users will need it in order to build the code, so it's best if it comes in the package.